manifests/agent.pp

```diff
@@ -134,6 +134,16 @@
 #    monitored_by_proxy => 'my_proxy_host',
 #    server             => '192.168.1.1',
 #  }
+#
+# @example Using Zabbix Agent 2
+#   class { 'zabbix::agent':
+#     agent_configfile_path => '/etc/zabbix/zabbix_agent2.conf',
+#     include_dir           => '/etc/zabbix/zabbix_agent2.d',
+#     pidfile               => '/var/run/zabbix/zabbix_agentd2.pid',
```

Comment:
is it actually required to pass the pidfile?

Reply:
Yes, default PID was on /tmp but for example on ubuntu22.04 unit file use different file